Objects/typeobject.c

```diff
@@ -7661,6 +7662,54 @@ wrap_descr_delete(PyObject *self, PyObject *args, void *wrapped)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+wrap_buffer(PyObject *self, PyObject *args, void *wrapped)
+{
+    int flags = 0;
+
+    if (!check_num_args(args, 1)) {
+        return NULL;
+    }
+    if (!PyArg_ParseTuple(args, "i", &flags)) {
```

Comment:
FWIW you can speed it up by directly converting it to int rather than going through slow `PyArg_ParseTuple`.